tests/a_scripts/data_prep/test_convert_text_to_mds.py

```diff
@@ -194,7 +193,8 @@ def call_convert_text_to_mds() -> None:
     n_tokens = 0
     for i in range(dataset.num_samples):
         sample = dataset[i]
-        tokens = np.frombuffer(sample['tokens'], dtype=int)
+        # TODO: delete the below
```

Comment:
whats the todo for?

Reply:
Oh i deleted the line and didn't delete the TODO lol